downloads/index.md

```diff
@@ -256,25 +278,21 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
 ~~~
 Most Linux users should use the glibc binaries unless you know that your system uses musl as its libc.
 
-## Juliaup - Julia version manager
-
-You can also install Julia with [juliaup](https://github.com/JuliaLang/juliaup) - a cross-platform Julia installer that makes working with multiple julia versions easy and fun!
-
-## Older Releases
+### Older Releases
 
 Older releases of Julia for all platforms are available on the [Older releases page](/downloads/oldreleases/). Only the LTS and Stable releases are maintained.
 
-## Nightly Builds
+### Nightly Builds
 
 Builds of the current unstable development version of Julia are available on the [nightlies page](/downloads/nightlies/). While the name suggests that they are built every night, they are actually built after each commit to master. However, at times due to broken builds or CI infrastructure issues, nightlies may not be produced. These are intended as developer previews into the latest work and are not intended for normal use. Most users are advised to use the current stable release version of Julia.
 
-## Download verification
+### Download verification
 
 All Julia binary releases are cryptographically secured using the traditional methods on each platform. macOS and Windows releases are codesigned with certificates that are verified by the operating system during installation. Linux and source tarballs are signed with GPG using [this key](/assets/juliareleases.asc).
 
 ---
 
-## Supported platforms
+### Supported platforms
```

Comment:
Perhaps this should go above the manual download links

Reply:
I gave this a go but I think it's helpful for the version history to be within the viewpane so that it's still easy to see what the current version is.